src/Nfts/Nfts.tsx

```diff
@@ -10,15 +10,21 @@ import {useNavigateTo} from './navigation'
 import {NoNftsScreen} from './NoNftsScreen'
 
 export const Nfts = () => {
-  const {search, nfts, isLoading, refetch, isRefetching, isError} = useFilteredNfts()
+  const {search, nfts, isLoading, refetch, isError} = useFilteredNfts()
```

Comment:
`isRefetching` | `isFecthing` won't work here?

Reply:
both have same values for `refetch` for manual and automatic actions, so its not possible differentiating the origin of the action